utils/walk.js

```diff
@@ -0,0 +1,45 @@
+const bcd = require('..');
+const { isBrowser, descendantKeys, joinPath } = require('./walkingUtils');
+const query = require('./query');
+
+function* lowLevelWalk(data = bcd, path, depth = Infinity) {
+  if (path !== undefined) {
+    yield {
+      path,
+      data,
+      compat: data.__compat,
```

Comment:
How about including only one of `compat` and `browser`, rather than having one be undefined? Trying this out and seeing `browser: undefined` I first wondered when `browser` would ever not be null, if there's a browser filter or something. Not so :)

Reply:
Oh, yep, that's a reasonable interpretation of the undefined values. I've pushed a change that sets `browser`, `compat`, or neither, but never both.